ui_testing/testcases/basic_tests/test04_testunits.py

```diff
@@ -851,3 +851,90 @@ def test030_allow_unit_field_to_be_displayed_in_case_of_mibi(self):
             fixed_sheet_row_data = self.fix_data_format(values)
             self.base_selenium.LOGGER.info('search for value of the unit field: {}'.format(row_data['Unit']))
             self.assertIn(row_data['Unit'], fixed_sheet_row_data)
+
+    def test031_editing_limit_of_quantification_fields_should_affect_table_and_version(self):
+        """
+        New: Test unit: Limits of quantification Approach: Versions:
+        In case I edit any field in the limits of quantification and press on save and create new version,
+        new version should create & display in the active table & versions table
+
+        When I edit any field in limits of quantification and press on save and create new version ..new version created 
+        This version should displayed in the active/archive table 
+        This version should display in the versions table 
```

Comment:
duplicate

Reply:
@0xIslamTaha is it duplicated for Limits of quantification? or duplicated checking in version in general ?

Reply:
You wrote this string twice